2025-10-14 01:00:02,072 INFO [App]: Запуск работы «wb_control_service»
2025-10-14 01:00:02,075 INFO [DBClient]: Таблица positions уже существует
2025-10-14 01:00:02,075 INFO [ApiClient]: Запрос геоданных с API WB
2025-10-14 01:02:19,015 ERROR [App]: Возникла ошибка: HTTPSConnectionPool(host='nominatim.openstreetmap.org', port=443): Max retries exceeded with url: /search?q=%D0%9A%D0%B0%D0%B7%D0%B0%D0%BD%D1%8C&format=json&limit=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7ea823aa2630>: Failed to establish a new connection: [Errno 101] Network is unreachable')). Завершение работы
2025-10-14 01:02:19,015 INFO [DBClient]: Соединение с базой данных закрыто
2025-10-14 01:02:19,015 INFO [App]: Завершение работы «wb_control_service»
2025-10-14 01:02:19,016 INFO [App]: Загрузка завершена за 136.94 секунд
2025-10-14 01:02:19,016 INFO [App]: ======================================================================

